src/cmd-cloud-prune

```diff
@@ -414,5 +444,63 @@ def prune_build(s3_client, bucket, prefix, build_id, dry_run):
                 raise Exception(f"Error pruning {build_id}: {e.response['Error']['Message']}")
 
 
+def get_container_tags(meta_json, exclude):
+    base_oscontainer = meta_json.get("base-oscontainer")
+    if base_oscontainer:
+        tags = base_oscontainer.get("tags", [])
+        filtered_tags = [tag for tag in tags if tag not in exclude]
+        container_repo = base_oscontainer.get("image", "")
+        return filtered_tags, container_repo
+    return [], ""
+
+
+def prune_container(tag, dry_run, container_repo, registry_auth_file):
+    if dry_run:
+        print(f"Would prune image {container_repo}:{tag}")
+    else:
+        skopeo_delete(container_repo, tag, registry_auth_file)
+
+
+def get_update_graph(stream):
+    url = f"https://builds.coreos.fedoraproject.org/updates/{stream}.json"
+    r = requests.get(url, timeout=5)
+    if r.status_code != 200:
+        raise Exception(f"Could not download update graph for {stream}. HTTP {r.status_code}")
+    return r.json()
+
+
+def skopeo_inspect(repo, image, auth):
```

Comment:
OPTIONAL:

For consistency it might make it better to use `tag` as the variable name in this function too:

```suggestion
def skopeo_inspect(repo, tag, auth):
```

Reply:
Added this for consistency